app/views/candidate_interface/course_choices/offer.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, t('page_titles.view_and_respond_to_offer') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_complete_path, 'Back to application dashboard') %>
+
+<h1 class="govuk-heading-xl govuk-heading-xl">
+  <span class="govuk-caption-xl"><%= @course_choice.provider.name %> - <%= @course_choice.course_option.course.name %> (<%= @course_choice.course_option.course.code %>)</span>
+  <%= t('page_titles.view_and_respond_to_offer') %>
+</h1>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render(OfferReviewComponent, course_choice: @course_choice) %>
+
+    <p class="govuk-body">Remember:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>you are under no obligation to accept or decline an offer at this stage</li>
+      <li>you can only accept one offer, across both Apply for teacher training and UCAS Teacher Training</li>
+    </ul>
+
+    <h2 class="govuk-heading-xl govuk-heading-m">
+      How do you want to respond to this offer?
+    </h2>
+
+    <p class="govuk-body">You can respond to offer to study:</p>
```

Comment:
I don't think this line reads right?

Reply:
You're right @fofr! 🤦‍♀ Changed a few words:

![image](https://user-images.githubusercontent.com/42817036/69526571-5e358980-0f62-11ea-8fc2-595b2284efb1.png)

How's this?

Reply:
Updated the content of this with @laurahermionetennant92 to:

![image](https://user-images.githubusercontent.com/42817036/69536241-8aa6d100-0f75-11ea-9a59-840c9954b6bb.png)

Reply:
🙌